egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -478,13 +603,16 @@ def GetLearningRate(iter, num_jobs, num_iters, num_archives_processed,
 
     return num_jobs * effective_learning_rate
 
-def DoShrinkage(iter, model_file, non_linearity, shrink_threshold):
+def DoShrinkage(iter, model_file, name, non_linearity, shrink_threshold, use_raw_nnet = False):
```

Comment:
@freewym Could you please run a simple experiment where you disable shrinkage for RNN training. IIRC it did not lead to WER improvements when I last tested this. It was just ensuring good gradient means, and we have other methods like self-repair to ensure this right now.